app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardCSVStep/CSVDropzoneField.js

```diff
@@ -77,52 +96,82 @@ class CSVDropzoneField extends React.Component {
       <ListView>
         {value.map(vm => (
           <ListView.Item
+            key={vm.reference}
             description={vm.name || ''}
             additionalInfo={[
-              <ListView.InfoItem>
+              <ListView.InfoItem key="provider">
                 {`${__('Provider:')} ${vm.provider || ''}`}
               </ListView.InfoItem>,
-              <ListView.InfoItem>
+              <ListView.InfoItem key="reference">
                 {`${__('Reference:')} ${vm.reference || ''}`}
               </ListView.InfoItem>
             ]}
           />
         ))}
       </ListView>
     );
-    const results = csvDropped && (
+    const results = csvImported && (
       <Toolbar.Results>
         {!error && value && sprintf(__('%s Items'), value.length)}
         {pristine && error}
       </Toolbar.Results>
     );
     return (
-      <Dropzone
-        accept=".csv"
-        className={cx('csv-upload-dropzone')}
-        activeClassName={cx('active')}
-        onDrop={this.onFileDrop}
-        disableClick
-        disablePreview
-        ref={node => {
-          dropzoneRef = node; // See comment at top, this is so we can call dropzoneRef.open below
-        }}
-      >
-        <Toolbar>
-          <Toolbar.RightContent>
-            <Button
-              bsStyle="primary"
-              onClick={() => {
-                dropzoneRef.open();
-              }}
-            >
-              {__('Browse...')}
-            </Button>
-          </Toolbar.RightContent>
-          <div className="form-group">{mainContents}</div>
-          {results}
-        </Toolbar>
-      </Dropzone>
+      <React.Fragment>
+        <Dropzone
+          accept=".csv"
+          className={cx('csv-upload-dropzone')}
```

Comment:
can we abbreviate to `className="csv-upload-dropdown"`?

Reply:
So I am realizing now that I may have been mistaken about `classNames`/`cx` doing namespacing with the sass classes. That may have been specific to my last team's configuration... @priley86 does our `cx` do namespacing? so this class's styles wouldn't affect any other components? if not yeah I agree we can just use the raw string. @michaelkro I may have led you astray here haha

Reply:
yea - we haven't used it for namespacing yet... just conditionals...but that is good to know though  👍

Reply:
@michaelkro take note here--- i was wrong. `classNames` does not namespace things by default.